src/main/java/uk/gov/hmcts/darts/common/repository/SecurityRoleRepository.java

```diff
@@ -4,11 +4,14 @@
 import org.springframework.stereotype.Repository;
 import uk.gov.hmcts.darts.common.entity.SecurityRoleEntity;
 
+import java.util.List;
 import java.util.Optional;
 
 @Repository
 public interface SecurityRoleRepository extends JpaRepository<SecurityRoleEntity, Integer> {
 
     Optional<SecurityRoleEntity> findByRoleName(String string);
 
+    List<SecurityRoleEntity> findAllByOrderById();
```

Comment:
this name sounds weird - is it right?

Reply:
yes, its in the standard JPA format, if there was a mistake, it wouldn't work as the name infers the functionality.
See here for reference
https://stackoverflow.com/questions/25486583/how-to-use-orderby-with-findall-in-spring-data